packages/cashier/src/modules/deposit-fiat/components/deposit-fiat-iframe/deposit-fiat-iframe.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { Loading } from '@deriv/components';
+import { useDepositFiatAddress } from '@deriv/hooks';
+import { observer } from '@deriv/stores';
+import './deposit-fiat-iframe.scss';
+
+const DepositFiatIframe: React.FC = observer(() => {
+    const { data: iframe_url, isSuccess } = useDepositFiatAddress();
+    const [is_loading, setIsLoading] = useState(true);
+
+    React.useEffect(() => {
+        setIsLoading(true);
+    }, [iframe_url]);
+
+    return (
+        <React.Fragment>
+            {is_loading && <Loading className='deposit-fiat-iframe__loader' is_fullscreen={false} />}
```

Comment:
```suggestion
            {is_loading && <Loading is_fullscreen={false} />}
```
@aum-deriv The class name doesn't exist anymore.